docs/transparent-validation/sd-wire/usage-validation.md

```diff
@@ -20,7 +20,11 @@ To prepare the environment reproduce the following steps:
        terminal and type the following command:
 
         ```bash
+        # Ubuntu/Debian:
         dpkg -L libftdi1-dev
+
```

Comment:
@Psotas could we separate these instructions? Currently, if we want to copy these instruction, we are forced to copy for both distribution types.